db/migrate/20160301070319_add_version_ostree_branches.rb

```diff
@@ -0,0 +1,30 @@
+class AddVersionOstreeBranches < ActiveRecord::Migration
+  def up
+    drop_table :katello_ostree_branches if ActiveRecord::Base.connection.table_exists? "katello_ostree_branches"
+
+    create_table :katello_ostree_branches do |t|
```

Comment:
We have to drop and then reindex anyway so dropping the table and recreating it seems easier than dropping all rows and then migrating the table. Either works for me though.